src/screens/DiscoveryScreenV2/fragment/UsersFragment.js

```diff
@@ -298,21 +298,21 @@ const UsersFragment = ({
 const styles = StyleSheet.create({
   fragmentContainer: {
     flex: 1,
-    backgroundColor: COLORS.white
+    backgroundColor: COLORS.almostBlack
   },
   noDataFoundContainer: {
     flex: 1,
-    backgroundColor: COLORS.white,
+    backgroundColor: COLORS.almostBlack,
     justifyContent: 'center'
   },
   noDataFoundText: {
     alignSelf: 'center',
     justifyContent: 'center',
     fontFamily: fonts.inter[600],
-    color: COLORS.white2
+    color: COLORS.white
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The text color in `noDataFoundText` has been updated from `COLORS.white2` to `COLORS.white`. This might affect the visibility of the text against its new background color (`COLORS.almostBlack`). Please ensure that this change provides sufficient contrast for readability.
```diff
-    color: COLORS.white
+    color: COLORS.white2
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->